content/docs/handling-events.md

```diff
@@ -91,64 +92,68 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/xEmzGg?editors=0010)
+[**Փորձել CodePen-ում**](https://codepen.io/anon/pen/mommWK?editors=0010)
+
+JSX-ի հետկանչում ուշադիր եղեք `this`-ի նշանակությանը։ JavaScript-ում կլասի մեթոդները իսկզբանե [կցված](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Function/bind) չեն։ Եթե դուք մոռանաք կցել `this.handleClick`-ը և տեղադրել դա `onClick`-ում, `this`-ի արժեքը կլինի `undefined` երբ ֆունկցիան կանչվի։
```

Comment:
default - ը իսկզբանեն չի։ default value - լռությամբ արժեք։ class methods are not bound by default -> կլասի մեթոդները լռությամբ կցված չեն կոնտեքստին։

Reply:
done